library/std/src/sys_common/rt.rs

```diff
@@ -38,8 +39,21 @@ pub fn cleanup() {
     });
 }
 
+macro_rules! rterr {
```

Comment:
The name could be something different, `rtwrite`, `rtprint`, `rtoutput` etc.

Reply:
Maybe something with `print` in the name would be better. `err` might suggest it returns from the function or panics or aborts.

Reply:
`rtprintpanic!`? I think I'll also add a comment about that this writes to the designated "panic output", which is just `stderr` on most platforms.